verify_rpms/rpm_verifier.py

```diff
@@ -163,28 +164,21 @@ def __call__(self, img: str) -> ProcessedImage:
     type=click.Path(path_type=Path),
     required=True,
 )
-@click.option(
-    "--status-path",
-    help="Path in which the status will be written to",
-    type=click.Path(path_type=Path),
-    required=True,
-)
 def main(
     img_input: str,
     fail_unsigned: bool,
     workdir: Path,
-    status_path: Path,
 ) -> None:
     """Verify RPMs are signed"""
-
+    status_path: Path = Path(os.path.join(workdir, "status"))
```

Comment:
By doing it, it also fix the comment for unspecified-encoding.
Win WIn :)

Reply:
:)

I was thinking that might be it. I think the reason that you're getting this is because status_path coming from click had some default encoding assigned to it, but with using os you're creating a new Path object which seems not to have encoding associated with it so you're seeing this error.